bin/found_neoepitopes.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+"""
+Commandline tool for extracting unique neoepitopes generated from MHCFlurry predictions
+and the FRED2 vcf_neoepitope_predictor.py script.
+
+usage: found_neoepitopes.py [-h]
+                                -n NEOEPITOPES
+                                -p PREDICTED_PEPTIDES
+                               [-f FILEFORMAT {raw, csv, json}]
+                                -o OUTPUT
+
+Neoepitope prediction for TargetInsepctor.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -n, --neoepitopes NEOEPITOPES     Path to the neoepitopes input file
+  -p, --predicted_peptides PEPTIDES
+                        Path to numerous files containing predicted peptides by MHCFlurry
+  -f, --file_format {raw, csv, json}
+                        File format to report result in
+  -o, --output OUTPUT
+                        Path to the output file
+"""
+import argparse
+import csv
+import json
+import logging
+import re
+import sys
+from itertools import tee
+from collections import defaultdict
+
+# logging setup
+console = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+console.setFormatter(formatter)
+LOG = logging.getLogger("Found Neoepitopes")
+LOG.addHandler(console)
+LOG.setLevel(logging.INFO)
+
+
+def parse_vcf_neoepitopes(neoepitope_file, alleles):
+    """
+    parses the output of the VCF neoepitope predictor script
+
+    :param neoepitope_file: output file of VCF neoepitope script
+    :param alleles: all alleles which were used for the VCF neoepitope script
+    :return: dictionary of alleles to peptide sequences
+    """
+    HLA_allele_to_peptides = defaultdict(list)
```

Comment:
Nice work, but I wonder whether it might be better to do that independent of the allele